### PR TITLE
fixed metadata bug

### PR DESCRIPTION
**Pull request by NikitaPoly**

### CODE REVIEW

**Comment by tannermpeterson** [comment]
What bug is this fixing?

**Comment by NikitaPoly** [comment]
> What bug is this fixing?

This bug is from new files that Eric tried using on the dash board. Originally he sent me one file and I added support for them. This bug was present when Eric tried to run a different optical file, ones he had not sent to me previously. They failed during pulse3d analysis so this is the fix for those files. I ran all three files and was able to get an output xlsx.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,13 @@
 Changelog for Pulse3D
 =====================
+0.32.2 (2020-03-23)
+-------------------
+
+Fixed:
+^^^^^^
+- Fixed missing wells when reading metadata for optical files.
```
Comment (tannermpeterson) [inline CHANGELOG.rst:8]: This should say something more along the lines of not trying to grab metadata from wells that aren't present when loading from optical files

**Review by tannermpeterson — CHANGES_REQUESTED** [review]

File: tests/xlsx/test_excel_writer.py [hunk 2]
```diff
@@ -111,13 +113,16 @@ def test_write_xlsx__runs_magnet_finding_alg_without_error(patch_get_positions):
 
 
 @pytest.mark.slow
-def test_write_xlsx__runs_optical_file_without_error():
+@pytest.mark.parametrize(
```
Comment (tannermpeterson) [inline tests/xlsx/test_excel_writer.py:115]: 👍

**Review by tannermpeterson — APPROVED** [review]

**Pull request merged**